src/OmniaMigrationTool/Services/ExportService.cs

```diff
@@ -535,6 +537,10 @@ object Map(object value)
                         default:
                             if (attribute.Target.Equals("_code"))
                                 return value.ToString().Substring(0, Math.Min(31, value.ToString().Length)); // TODO: Deal the the difference of size in codes
+                            // Remove spaces from the IncidentCategory code
+                            if (attribute.Target.Equals("IncidentCategory"))
```

Comment:
This change shouldn't be part of this project, since it seems a scenario specific from your application.